src/graph/backend/dnnl/patterns/conv_post_ops.cpp

```diff
@@ -32,8 +32,11 @@ namespace {
 bool check_s8_weight(op_t *op) {
     const op_kind_t kind = op->get_kind();
     if (kind != graph::op_kind::Dequantize) return true;
-    return (!is_int8_quantization(op)
-            || check_input_dtype<graph::data_type::s8>(op));
+    bool result = !is_int8_quantization(op)
+            || check_input_dtype<graph::data_type::s8>(op);
+    VCHECK_PATTERN_UTILS(
+            result, result, "conv primitive only support s8 weight");
```

Comment:
```suggestion
            result, result, "conv primitive only supports s8 weight");
```

I also see grammar mistakes in other log messages. Please check them.

Reply:
Thanks. I fixed some msg bugs. Could you take another look?